clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleLoadBalancerDescriptionValidatorSpec.groovy

```diff
@@ -126,4 +155,200 @@ class UpsertGoogleLoadBalancerDescriptionValidatorSpec extends Specification {
       1 * errors.rejectValue('loadBalancerName', _)
       1 * errors.rejectValue('region', _)
   }
+
+  void "pass validation with proper http description inputs"() {
+    setup:
+    def input = [
```

Comment:
Up to this point, this file has the content of the `setup`/`when`/`then` blocks indented.